homework/kostyantyn-gamanyuk/06-lesson/gulpfile.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+//определим переменные для функцийи плагинов
```

Comment:
all comments on the code should be written in English, you do not need to change them in this file, but in future works write only in English